DEFINITION_OF_DONE.md

```diff
@@ -0,0 +1,20 @@
+# Definition of Done (DoD)
+
+This document is a collaborative effort among team members to establish clear criteria for completing tasks. The DoD serves as a guide to ensure the quality and completeness of work within the team.
+
+To meet our standards, every task should adhere to the following criteria:
+
+- Code has been reviewed by at least one other team member.
+- Functionality works as expected and acceptance criteria are met
+- Coding standards are followed.
+- No critical issues or bugs are present.
```

Comment:
I'm not so sure about this one:
> No critical issues or bugs are present.

Even with unit tests, bugs may still be present. Unit tests can be used to test for the presence of bugs, but not their absence. 🥲

Reply:
I agree. In my opinion `No critical issues are present` would be better, in a sense that would simply be : The lib is not crashing immediately.,

Reply:
Has been changed to : `No critical issues are presen`.